test_mavros/CMakeLists.txt

```diff
@@ -79,10 +86,10 @@ find_package(catkin REQUIRED COMPONENTS
 ## CATKIN_DEPENDS: catkin_packages dependent projects also need
 ## DEPENDS: system dependencies of this project that dependent projects also need
 catkin_package(
-#  INCLUDE_DIRS include
-#  LIBRARIES test_mavros
-#  CATKIN_DEPENDS mavros mavros_extras
-#  DEPENDS system_lib
+  INCLUDE_DIRS include
```

Comment:
For test usually there nothing. It is not normal package, so no reason to export depends.